README.md

```diff
@@ -67,7 +73,7 @@ Select 1:
 [] Selects all li's which are directly inside a ul of class dropdown (children)
```

Comment:
The Answer to Question #5 is actually the first: ` [X] Selects all li's which are directly inside a ul of class dropdown (children)`. 

It's selecting all children `<li>` of the `<ul> class of "dropdown"`